src/sentry/tasks/clear_expired_invites.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, print_function
+
+from datetime import timedelta
+
+from django.utils import timezone
+
+from sentry.models import OrganizationMember
+from sentry.tasks.base import instrumented_task
+
+
+@instrumented_task(name='sentry.tasks.clear_expired_invites')
+def clear_expired_invites():
+    """
+    Delete organization member invitations that have not been claimed
+    and have been open for more than 90 days.
+    """
+    ninety_days = timezone.now() - timedelta(days=90)
+    query = OrganizationMember.objects.filter(
+        token_expires_at__lt=ninety_days,
```

Comment:
We also would need an index here.

Reply:
Also, just checking, barely any rows have this column set. What do we do with the ones that are NULL?

Reply:
Sorry for the spam, shooting from the hip here.

Since the column is `token_expires_at` doesn't this imply that we wouldn't be checking 90 days, and we'd always do `token_expires_at__lt=timezone.now()` ?

Reply:
I was thinking of only purging records 90 days after they had expired. We can remove entries more eagerly though.

> Also, just checking, barely any rows have this column set. What do we do with the ones that are NULL

The rows with NULL predate my changes to add expirations. There are a few options that I can see:

1. Backfill all the existing rows with an expiration date based on date_added. This will allow the purge task to incrementally purge them.
2. In late January of next year we could delete all rows with token_expires_at=NULL as they would have hit their 90 days by that point.

Personally I lean towards 2 as it is less work to build a one time SQL query/script vs. a migration to backfill all the rows.